src/OrbitGl/CallTreeView.cpp

```diff
@@ -230,10 +265,12 @@ std::unique_ptr<CallTreeView> CallTreeView::CreateTopDownViewFromPostProcessedSa
   return current_node;
 }
 
-[[nodiscard]] static CallTreeUnwindErrors* AddUnwindErrorToBottomUpViewAndReturnUnwindingErrorsNode(
-    CallTreeView* bottom_up_view, const CallstackInfo& resolved_callstack,
-    uint64_t callstack_sample_count, const ModuleManager& module_manager,
-    const CaptureData& capture_data) {
+[[nodiscard]] static CallTreeUnwindErrorType*
+AddUnwindErrorToBottomUpViewAndReturnUnwindingErrorTypeNode(CallTreeView* bottom_up_view,
```

Comment:
`AddUnwindErrorToBottomUpViewAndReturnUnwindErrorTypeNode` (it was also wrong before)